src/colour-names-stub.hh

```diff
@@ -0,0 +1,28 @@
+/*
+ * To generate colour-names.hh, you must have gperf installed during build.
+ * This is a dummy implementation for builds without gperf.
+ * Color name matching will always return null (i.e. no match).
+ */
+
+#pragma once
+
+#include <cstdint>
+
+#include "logging.h"
+
+struct rgb {
+  const char *name;
+  uint8_t red;
+  uint8_t green;
+  uint8_t blue;
+};
+
+class color_name_hash {
+ public:
+  static const struct rgb *in_word_set(const char *str, size_t len);
+};
+
+const struct rgb *color_name_hash::in_word_set(const char *str, size_t len) {
+  DBGP("color parsing not supported");
```

Comment:
I guess the only thing that I'm not happy with in my initial impl is that this should've been `NORM_ERR` and placed last (after hex parser).
That would've avoided confusion in that one issue.

But given that the option is enabled by default, it's fine I guess. Maybe make it `DBGP2` because it will spam a lot.